src/client/src/components/pages/participants/addParticipant/index.jsx

```diff
@@ -11,13 +11,24 @@ import axios from "axios";
 import swal from "sweetalert2";
 
 export default class index extends Component {
-  state = initialState;
+  state = initialState
+  
 
   onChange = event => {
     const { value, name } = event.target;
-    this.setState({ [name]: value });
+      this.setState({ [name]: value });
   };
 
+  // check the participant in prison or not
+  onChecked = event => {
+    if (this.state.checked) {
+      event.target.style.background = '#ccc'
+    } else {
+      event.target.style.background = '#FF4800'
+    }
+    this.setState({ checked: !this.state.checked });
+  }
```

Comment:
@mohannadhanafi I think it would be a nice improvement if you could move this color change into render() or into the `<Form>` - then your UI is changing in response to the state change.
This is a cleaner design and would keep the UI in sync better with state.  For example, if checked was initially true (i.e. if `initialState.checked` was true), then the current logic with the color change inside the event handler would mean that the checkbox was improperly colored initially, but moving the color logic inside render() would mean that this initial situation would be displayed properly.